save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/topbar/TopBarUserField.kt

```diff
@@ -50,7 +50,7 @@ val topBarUserField: FC<TopBarUserFieldProps> = FC { props ->
     val navigate = useNavigate()
     var isLogoutModalOpen by useState(false)
     var isAriaExpanded by useState(false)
-    val (avatar, setAvatar) = useStateFromProps("/api/$v1/avatar${props.userInfo?.avatar}")
+    val (avatarNew, setAvatarNew) = useStateFromProps(props.userInfo?.avatar?.let { "/api/$v1/avatar/$it" } ?: AVATAR_PROFILE_PLACEHOLDER)
```

Comment:
btw, this fixed 404 in some cases

Reply:
Can also be written like
```Kotlin
    val (avatarNew, setAvatarNew) = useStateFromProps(props.userInfo) { it?.avatar?.let { "/api/$v1/avatar/$it" } ?: AVATAR_PROFILE_PLACEHOLDER }
```

if I am not mistaken

just in case of lack of convenience

Reply:
feels less readable and Kotlin style :)

Reply:
oh ok